src/web/src/config.ecpp

```diff
@@ -38,64 +38,378 @@
+BIOS_SMTP_USE_AUTHENTICATION = ""
+    filepath = "/etc/fty-email/fty-email.cfg"
+    treepath = "smtp/use_auth"
+
+# agent-ms
+BIOS_METRIC_STORE_AGE_RT = ""
+    filepath = "/etc/fty-metric-store/fty-metric-store.cfg"
+    treepath = "store/rt"
+BIOS_METRIC_STORE_AGE_15m = ""
+    filepath = "/etc/fty-metric-store/fty-metric-store.cfg"
+    treepath = "store/15m"
+BIOS_METRIC_STORE_AGE_30m = ""
+    filepath = "/etc/fty-metric-store/fty-metric-store.cfg"
+    treepath = "store/30m"
+BIOS_METRIC_STORE_AGE_1h = ""
+    filepath = "/etc/fty-metric-store/fty-metric-store.cfg"
+    treepath = "store/1h"
+BIOS_METRIC_STORE_AGE_8h = ""
+    filepath = "/etc/fty-metric-store/fty-metric-store.cfg"
+    treepath = "store/8h"
+BIOS_METRIC_STORE_AGE_24h = ""
+    filepath = "/etc/fty-metric-store/fty-metric-store.cfg"
+    treepath = "store/24h"
+BIOS_METRIC_STORE_AGE_7d = ""
+    filepath = "/etc/fty-metric-store/fty-metric-store.cfg"
+    treepath = "store/7d"
+BIOS_METRIC_STORE_AGE_30d = ""
+    filepath = "/etc/fty-metric-store/fty-metric-store.cfg"
+    treepath = "store/30d"
+
+# fty-discovery
+FTY_DISCOVERY_TYPE = ""
+    filepath = "/etc/fty-discovery/fty-discovery.cfg"
+    treepath = "discovery/type"
+FTY_DISCOVERY_SCANS = ""
+    filepath = "/etc/fty-discovery/fty-discovery.cfg"
+    treepath = "discovery/scans"
+FTY_DISCOVERY_IPS = ""
+    filepath = "/etc/fty-discovery/fty-discovery.cfg"
+    treepath = "discovery/ips"
+FTY_DISCOVERY_DOCUMENTS = ""
+    filepath = "/etc/fty-discovery/fty-discovery.cfg"
+    treepath = "discovery/documents"
+FTY_DISCOVERY_DEFAULT_VALUES_STATUS = ""
+    filepath = "/etc/fty-discovery/fty-discovery.cfg"
+    treepath = "discovery/defaultValuesAux/status"
+FTY_DISCOVERY_DEFAULT_VALUES_PRIORITY = ""
+    filepath = "/etc/fty-discovery/fty-discovery.cfg"
+    treepath = "discovery/defaultValuesAux/priority"
+FTY_DISCOVERY_DEFAULT_VALUES_PARENT = ""
+    filepath = "/etc/fty-discovery/fty-discovery.cfg"
+    treepath = "discovery/defaultValuesAux/parent"
+FTY_DISCOVERY_DEFAULT_VALUES_LINK_SRC = ""
+    filepath = "/etc/fty-discovery/fty-discovery.cfg"
+    treepath = "discovery/defaultValuesLinks/0/src"
+
+# fty-discovery parameters
+FTY_DISCOVERY_DUMP_POOL = ""
+    filepath = "/etc/fty-discovery/fty-discovery.cfg"
+    treepath = "parameters/maxDumpPoolNumber"
+FTY_DISCOVERY_SCAN_POOL = ""
+    filepath = "/etc/fty-discovery/fty-discovery.cfg"
+    treepath = "parameters/maxScanPoolNumber"
+FTY_DISCOVERY_SCAN_TIMEOUT = ""
+    filepath = "/etc/fty-discovery/fty-discovery.cfg"
+    treepath = "parameters/nutScannerTimeOut"
+FTY_DISCOVERY_DUMP_LOOPTIME = ""
+    filepath = "/etc/fty-discovery/fty-discovery.cfg"
+    treepath = "parameters/dumpDataLoopTime"
+
+# fty-session
+FTY_SESSION_TIMEOUT_NO_ACTIVITY = ""
+    filepath = "/etc/fty/fty-session.cfg"
+    treepath = "timeout/no_activity"
+FTY_SESSION_TIMEOUT_LEASE = ""
+    filepath = "/etc/fty/fty-session.cfg"
+    treepath = "timeout/lease_time"
+
+# IPM2
```

Comment:
IPM2? inside fty ?

Reply:
Ideally this data would be split across files in a directory where each component declares its mapping, but now's not the time to do large-scale refactoring across multiple repositories (this needs to be backported to the 2.1.0 release).